vat.go

```diff
@@ -15,8 +15,17 @@ Get VAT rate that is currently in effect for a given country
 */
 package vat
 
+import "time"
+
 // ViesLookupService is the interface for the VIES VAT number validation service
 var ViesLookupService LookupServiceInterface = &viesService{}
 
 // UKVATLookupService is the interface for the UK VAT number validation service
 var UKVATLookupService LookupServiceInterface = &ukVATService{}
+
+var serviceTimeout = time.Duration(60) * time.Second
```

Comment:
you can just do `var serviceTimeout =  time.Second * 60` 🤔

Reply:
You are correct. This was basically just copy-paste and I didn't really think about it